cli/lib/otel/index.ts

```diff
@@ -1,15 +1,19 @@
-import { CollectorTraceExporter, CollectorMetricExporter } from '@opentelemetry/exporter-collector'
+import { JaegerExporter } from '@opentelemetry/exporter-jaeger'
+import { PrometheusExporter } from '@opentelemetry/exporter-prometheus'
```

Comment:
Export via OTLP over HTTP instead (you will need to install `@opentelemetry/exporter-metrics-otlp-http`)

```suggestion
import { OTLPMetricExporter } from '@opentelemetry/exporter-metrics-otlp-http'
```